src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -44,13 +38,24 @@ export const SiteLaunchChecklistTitle = (): JSX.Element => {
 }
 
 const getTextProps = (index: number, tasksDone: number): TextProps => {
-  const disabled = tasksDone >= index + 1 || tasksDone < index - 1
+  const disabled = tasksDone < index - 1
```

Comment:
could i get more context on this change? previously, the text showed enabled styling if `tasksDone` was in the interval `[index - 1, index]` but this change causes the enabled styling to be `[index, ]`, which is a behavioural change.

Reply:
Yes, this is intended. The figma differentiates the colour of the "current" task, a "task" that is pending and a task that is already "done"